layouts/partials/htmlhead.html

```diff
@@ -11,4 +11,44 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
 		<link rel="stylesheet" href='{{ "assets/css/main.css" | absURL }}' />
 		<noscript><link rel="stylesheet" href='{{ "assets/css/noscript.css" | absURL }}' /></noscript>
+		 {{ range .Site.Params.custom_css }}
+        <link rel="stylesheet" href="{{ . | absURL }}">
+        {{ end }}
+		
+		<meta name="theme" content="hugo-theme-massively">
+		{{ .Hugo.Generator }}
```

Comment:
Is `.Hugo.Generator` this required? I think Hugo automatically adds this in now by default.

See line 5 of the current build:

view-source:https://curttimson.github.io/hugo-theme-massively/